### ISSUE TITLE
[Bug]Judgement of Blood (BB #695)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#695]
This issue was migrated from bitbucket.
**Original Reporter:** Flywithme
**Original Date:** 09.09.2014 15:35:08 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/695

<hr>

Judging [Seal of Blood](http://tbc.wowroad.info/?spell=31892) should return damage like priest's [Shadow Word: Death](http://tbc.wowroad.info/?spell=32996), with 1 second delay, currently it's instantly, so you can`t break CCs with it.

I mean Crowd Controls, in this case mainly poly, blind if you are lucky, it should simply work exactly like priest's shadow word death, except it doesn't reflect damage when the judgement gets absorbed, so sometimes its hard to break even polymorph if the mage has shields, but at this moment, Judgement with Seal of Blood deals damage to paladin instantly, so I can't break any polymorph.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Flywithme
**Original Date:** 09.09.2014 20:02:32 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/695/#comment-12254748

<hr>

Here's the video, as it should work:

https://www.youtube.com/watch?v=-RMpklNqd9A&list=UUpZR52wgEHlp7oW7M27LALw

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Alex_Step
**Original Date:** 19.11.2014 07:45:02 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/695/#comment-13736245

<hr>

This solves the problem:

Add in SpellMgr.cpp :

```
#!c++

+       case 32220:                      //  Judgement of Blood
+           spellInfo->speed = 5.0f;    // Minor delay
+           break;
        case 1833:                        // Cheap Shot
            spellInfo->speed = 1230.0f;   // Tiny delay
            break;
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 19.11.2014 15:00:44 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/695/#comment-13745935

<hr>

We talked about that, but cheers!

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 19.11.2014 17:10:13 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/695/#comment-13749291

<hr>

[Core/Spells] Add minor delay to Judgement of Blood.

```
          Thanks to Alex_Step.
          Fixes #695.
```

→ <<cset 5db78e25188a>>
